### PR TITLE
front: show 0s on output table when 0s has been added in input table

### LINKED ISSUE #8290: TSV2 : Output table doesn't show 0s stopping times
### What happened?

I created a train with a stopping time of 0s.
image

The train is created but I don't see my stopping time in the output table
image

### What did you expect to happen?

I expected to see the 0s stopping time in the output table

### How can we reproduce it (as minimally and precisely as possible)?

Create a train with a stopping time of 0s at any point

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Google Chrome

### OSRD version (top right corner `Account` button > `Informations`)

0a34296

### PR DESCRIPTION
**Pull request by theocrsb**
close #8290

### CODE REVIEW

File: front/src/modules/timesStops/helpers/scheduleData.ts [hunk 1]
```diff
@@ -30,6 +32,9 @@ export function formatScheduleData(scheduleData: ComputedScheduleEntry) {
   return {
     arrival: scheduleData.arrival ? secToHoursString(scheduleData.arrival, true) : '',
     departure: scheduleData.departure ? secToHoursString(scheduleData.departure, true) : '',
-    stopFor: scheduleData.stopFor ? String(scheduleData.stopFor) : '',
+    stopFor:
+      scheduleData.stopFor !== null && scheduleData.stopFor !== undefined
```
Comment (anisometropie) [inline front/src/modules/timesStops/helpers/scheduleData.ts]: `stopFor` is `number | null` you shouldn’t need to check for `undefined`

Comment (theocrsb) [reply front/src/modules/timesStops/helpers/scheduleData.ts]: You're right. It's done

File: front/src/modules/timesStops/helpers/scheduleData.ts [hunk 2]
```diff
@@ -30,6 +32,6 @@ export function formatScheduleData(scheduleData: ComputedScheduleEntry) {
   return {
     arrival: scheduleData.arrival ? secToHoursString(scheduleData.arrival, true) : '',
     departure: scheduleData.departure ? secToHoursString(scheduleData.departure, true) : '',
-    stopFor: scheduleData.stopFor ? String(scheduleData.stopFor) : '',
+    stopFor: scheduleData.stopFor !== null ? String(scheduleData.stopFor) : '',
```
Comment (Uriel-Sautron) [inline front/src/modules/timesStops/helpers/scheduleData.ts:35]: This solution solves the bug of 0s not being displayed in the output table, but we still have the problem we discussed together.
1. On the input table enter a value in one of the stop for.
2. Delete this value.
3. Add your train.
4. In the output table we find 0s in the stop for when it shouldn't be

**Review by Uriel-Sautron — APPROVED** [review]
LGTM, tested !!! Good work.

**Review by clarani — APPROVED** [review]
LGTM and tested :white_check_mark:

**Pull request merged**